common/djangoapps/microsite_configuration/templatetags/microsite.py

```diff
@@ -63,3 +63,14 @@ def microsite_css_overrides_file():
         return "<link href='{}' rel='stylesheet' type='text/css'>".format(static(file_path))
     else:
         return ""
+
+
+@register.filter
+def microsite_template_path(template_name):
+    """
+    Django template filter to apply template overriding to microsites.
+    The django_templates loader does not support the leading slash, therefore
+    it is stripped before returning.
+    """
+    template_name = microsite.get_template_path(template_name)
+    return template_name[1:] if template_name[0] == '/' else template_name
```

Comment:
Hi. I loaded up this branch on its devstack and tried to pass None in the template name. With or without the None guards it will throw an error, the difference would be if is either TemplateDoesNotExist: No exception supplied, or IndexError: string index out of range. I think it does not make a big difference, but if you think they are still required, just ping me here, and I will add them and push them right away.